### ISSUE TITLE
devterm-fan-temp-daemon running and generating errors

### ISSUE THREAD

**Issue by SvanteRichter** [clockworkpi/uConsole#5]
The systemd service devterm-fan-temp-daemon.service seems to generate errors every 5 seconds because it cannot find /opt/vc/bin/vcgencmd.

Its easily disabled, but I thought it might be good to disable it in the built images.

I assume its not needed since the uConsole does not have a fan to control, right?

**Comment by Mihaylov93** [comment]
Yeah, not only disabled but the python scripts also removed, as its Devterm specific
